data-loader/database_functions/fib_indices.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from flask import Flask
+from library import backend_util
+logger = logging.getLogger(__name__)
```

Comment:
For all of the scripts, can you please utilize the `get_logger` function that I have built into the package? See [here](https://github.com/mr-devs/top-FIBers/blob/e0e60f99f628b04b66910cb158ec550d448dd571/package/top_fibers_pkg/utils.py#L249).

This will ensure that we are logging everything in the same way. Also, as you're using the logger right now, (1) I don't believe that the logging messages are being recorded in a file that we'll be able to reference later. We will need this for when everything is automated by a cronjob. 

And (2) I don't think that the `logger.info` messages are being delivered (as the default logging level is WARNING [aka 30]). You can check this by sticking `print(logger.getEffectiveLevel())` right after this line. either way I think we should be consistent across the project.

Reply:
I added the logger to read the files and the output file will be in /home/data/apps/topfibers/repo/logs